semantic-core/src/Analysis/Eval.hs

```diff
@@ -39,7 +39,7 @@ eval :: ( Carrier sig m
      -> (Term (Ann Span :+: Core) Name -> m value)
 eval Analysis{..} eval = \case
   Var n -> lookupEnv' n >>= deref' n
-  Term (R c) -> case c of
+  Alg (R c) -> case c of
```

Comment:
`fused-syntax` gives the recursive constructor of `Term` the name `Alg` instead.